docs/classic-ui/images.md

```diff
@@ -254,81 +254,199 @@ The `ImageScaling` view explicitly checks the permissions of the current user.
 To access image scales, which are normally not accessible to the current user, override the `validate_access` method in `plone.namedfile.scaling.ImageScale`.
 
 
-## `srcset` configuration
+(classic-ui-images-responsive-image-support)=
+## Responsive image support
 
-In `/@@imaging-controlpanel` Plone allows you to define HTML {term}`srcset` attributes.
-A `srcset` can help the browser serve the best fitting image for the current users situation.
-Which image scale is best for the user can be decided on different metrics.
+Plone supports the generation of picture tags with `srcset`'s for image optimization.
+Additionally you can define `media queries` for {ref}`classic-ui-images-responsive-image-support-art-direction` and further optimization.
 
+The configuration allows to define different picture variants, like `Large`, `Medium`, `Small`.
+Users can choose from them in editors like TinyMCE and Developers can use them in templates.
 
-### viewport size
 
-The first is metric would be the viewport size.
+(classic-ui-images-responsive-image-support-picture-variants)=
+### Picture variants
+
+In `/@@imaging-controlpanel` Plone allows you to define picture variants with a list of available image scales.
+These are used for HTML {term}`srcset` attributes.
+A `srcset` attribute can help the browser to serve the best fitting image size for the current users situation.
+
+
+### default configuration
+
+The default configuration covers `image size optimization` and will provide the Browser with the needed information to load the optimal image size.
```

Comment:
I'm not sure why you put "image size optimization" in backticks. It is not an inline literal. Did you mean to define a Glossary term? That would be good.

```suggestion
The default configuration covers image size optimization and will provide the browser with the needed information to load the optimal image size.
```